dali/operators/decoder/audio/generic_decoder.cc

```diff
@@ -198,7 +198,7 @@ struct GenericAudioDecoder<SampleType>::Impl {
     ret.length = sf_info_.frames * sf_info_.channels;
     ret.channels = sf_info_.channels;
     ret.sample_rate = sf_info_.samplerate;
-    ret.channels_interleaved = false;
+    ret.channels_interleaved = true;
```

Comment:
I guess this info should be extracted from the file.

Reply:
We discussed it, I'll keep this hardcoded